worlds/_bizhawk/context.py

```diff
@@ -6,7 +6,7 @@
 import asyncio
 import enum
 import subprocess
-from typing import Any, Dict, Optional
+from typing import Any, Optional
```

Comment:
```suggestion
from typing import Any
```
Similar comments from client.py